msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -200,15 +201,23 @@ The following list details the various user types along with their accessibility
     > [!NOTE]
     > Anonymous users inherit the global default user-level app permission policy. For more information, see [manage Apps](/microsoftteams/non-standard-users#anonymous-user-in-meetings-access).
 
-A guest or anonymous user can't add, remove, or uninstall apps.
+A guest or anonymous user can't add, remove, or uninstall apps but they can *partially interact with the apps added by others.
```

Comment:
@vikramjit-singh-chug, sorry I missed this comment for doc update. But a guest can interact with apps in meeting and calls. I dont understand the intention of "*partial" keyword. 

We need to remove "Anonymous" user from the statement, since anonymous user cannot interact with apps unless the app is ⁠full trust apps with anonymous user access enabled in their manifest (currently Yammer is the only one).

Also @v-npaladugu, the note does not appear in Note section with correct formatting

Reply:
@v-npaladugu , please remove the line.